main.py

```diff
@@ -168,6 +174,29 @@ def load():
                 "flavour": expand(flavour), "starter": starter, "box": box,
                 "deck": deck, "start": start, "end": end
             })
+            # if not an unique card put into market dict too
+            if not "unique" in code and not starter:
+                if ctype == 'G':
+                    market_gems[casefold(name)].append({
+                        "name": name, "type": ctype, "cost": int(cost), "code": code,
+                        "special": expand(special, prefix=True), "text": expand(text),
+                        "flavour": expand(flavour), "starter": starter, "box": box,
+                        "deck": deck, "start": start, "end": end
+                    })
+                elif ctype == 'R':
+                    market_relics[casefold(name)].append({
+                        "name": name, "type": ctype, "cost": int(cost), "code": code,
+                        "special": expand(special, prefix=True), "text": expand(text),
+                        "flavour": expand(flavour), "starter": starter, "box": box,
+                        "deck": deck, "start": start, "end": end
+                    })
+                elif ctype == 'S':
+                    market_spells[casefold(name)].append({
+                        "name": name, "type": ctype, "cost": int(cost), "code": code,
+                        "special": expand(special, prefix=True), "text": expand(text),
+                        "flavour": expand(flavour), "starter": starter, "box": box,
+                        "deck": deck, "start": start, "end": end
+                    })
```

Comment:
No need for market_* dicts; the relevant code can check at the call site for relevant information.